### PR TITLE
Create ElastiCache client

### PR DESCRIPTION
**Pull request by Nyholm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,10 +19,11 @@ See full documentation on [https://async-aws.com](https://async-aws.com).
...
+| [async-aws/kinesis](https://github.com/async-aws/kinesis)                                     | [![Latest Stable Version](https://poser.pugx.org/async-aws/kinesis/v/stable)](https://packagist.org/packages/async-aws/kinesis)                                     [![Total Downloads](https://poser.pugx.org/async-aws/kinesis/downloads)](https://packagist.org/packages/async-aws/kinesis)                                     | [![](https://github.com/async-aws/kinesis/workflows/BC%20Check/badge.svg?branch=master)](https://github.com/async-aws/kinesisactions)                                          | [![](https://async-aws-pr.github.io/commits-since-release-counter/kinesis.svg)](https://github.com/async-aws/kinesis/releases)
```
Comment (Nyholm) [inline README.md:26]: Unrelated CI fix (whitespace)

File: src/CodeGenerator/src/Generator/GeneratorHelper.php [hunk 2]
```diff
@@ -70,6 +70,8 @@ public static function parseDocumentation(string $documentation, bool $short = t
             '<p>' => '',
             '<p/>' => '',
             '</p>' => "\n",
+            '<u>' => '',
+            '</u>' => '',
```
Comment (Nyholm) [inline src/CodeGenerator/src/Generator/GeneratorHelper.php:74]: I honestly did not know about this HTML tag. It was used in a comment here: https://github.com/async-aws/aws/pull/1087/files#diff-393723a3db6ca45a459be2595a3c6fb8195c8172fe11b9200f1bc91246b52beaR8

File: src/Service/CloudWatch/src/Result/GetMetricStatisticsOutput.php [hunk 3]
```diff
@@ -16,7 +16,7 @@ class GetMetricStatisticsOutput extends Result
     /**
      * The data points for the specified metric.
      */
-    private $datapoints = [];
+    private $datapoints;
```
Comment (Nyholm) [inline src/Service/CloudWatch/src/Result/GetMetricStatisticsOutput.php:19]: Unrelated CS changes

**Review by Nyholm — COMMENTED** [review]
The CS changes are fixed in #1088

File: src/CodeGenerator/src/Generator/GeneratorHelper.php (hunk 2 above)
Comment (jderusse) [reply src/CodeGenerator/src/Generator/GeneratorHelper.php:74]: good catch

File: src/Core/src/AwsClientFactory.php [hunk 4]
```diff
@@ -178,6 +179,19 @@ public function ecr(): EcrClient
         return $this->serviceCache[__METHOD__];
     }
 
+    public function elastiCache(): ElastiCacheClient
```
Comment (jderusse) [inline src/Core/src/AwsClientFactory.php:182]: Missing CHANGELOG entry.

Should we add a test in the CI for that?

**Review by jderusse — APPROVED** [review]

File: src/Core/src/AwsClientFactory.php (hunk 4 above)
Comment (Nyholm) [reply src/Core/src/AwsClientFactory.php:182]: Oh. Thank you. 

Hm... maybe. But I will not add that test now.
